test/specs/deployAndRetrieve.e2e.ts

```diff
@@ -88,10 +88,14 @@ describe('Deploy and Retrieve', async () => {
   });
 
   step('Deploy with SFDX: Deploy This Source to Org - ST enabled', async () => {
+    const workbench = await (await browser.getWorkbench()).wait();
     // Clear the Output view first.
     const outputView = await utilities.openOutputView();
-    await outputView.clearText();
-    const workbench = await (await browser.getWorkbench()).wait();
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Clear Output', 2);
```

Comment:
The "Clear Output" button in the Output panel often experiences flaky behavior - there is an "element not interactable" error if the button cannot be clicked.  Clicking the button can be replaced by running "View: Clear Output" in the command palette, which is much more stable.